File: msgraph-mail/go/utilities/users/mailFolders/message_rule_actions.go
package mailfolders
import (
    "github.com/microsoft/kiota-samples/msgraph-mail/go/utilities/users"
    "github.com/microsoft/kiota/abstractions/go/serialization"
)
type MessageRuleActions struct {
    additionalData Map<String, Object>;
    assignCategories []*string;
    copyToFolder *string;
    delete *bool;
    forwardAsAttachmentTo []Recipient;
    forwardTo []Recipient;
    markAsRead *bool;
    markImportance Importance;
    moveToFolder *string;
    permanentDelete *bool;
    redirectTo []Recipient;
    stopProcessingRules *bool;
}

File: msgraph-mail/swiftconsoleapp/Sources/swiftconsoleapp/Models/Microsoft/Graph/MessageRuleActions.swift
extension Swiftconsoleapp.Models.Microsoft.Graph {
public class MessageRuleActions : IAdditionalDataHolder {
public var additionalData: IDictionary<string, object>
public var assignCategories: [String]
public var copyToFolder: String
public var delete: Bool?
public var forwardAsAttachmentTo: [Recipient]
public var forwardTo: [Recipient]
public var markAsRead: Bool?
public var markImportance: Importance?
public var moveToFolder: String
public var permanentDelete: Bool?
public var redirectTo: [Recipient]
public var stopProcessingRules: Bool?
}
}
